### PR TITLE
feat: add JetBrains AI to readme.md

### PR DESCRIPTION
**Pull request by origamium**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ A list of of AI coding tools (assistants, completion, refactoring, etc.).
 - [GitLab Code Suggestions](https://docs.gitlab.com/ee/user/project/repository/code_suggestions.html)
 - [Sourcegraph Cody](https://about.sourcegraph.com/cody)
 - [Obsidian copilot auto completion](https://github.com/j0rd1smit/obsidian-copilot-auto-completion)
+- [Jetbrains AI](https://www.jetbrains.com/ai/)
```
Comment (jdorfman) [inline README.md:24]: Should be "Jet**B**rains"

**Review by jdorfman — CHANGES_REQUESTED** [review]
Looks good, just fix the B and I'll merge it :)

File: README.md (hunk 1 above)
Comment (origamium) [reply README.md]: Ok. I fixed that.

**Review by jdorfman — APPROVED** [review]
Thank you!

**Pull request merged**